### ISSUE TITLE
[Feature Request]: Creation of Signup Page

### ISSUE THREAD

**Issue by Saransh-Jainbu** [mansiruhil13/Bobble-AI#877]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Feature Description

# Feature Request: Signup Page Implementation

## Description

Currently, the site lacks a signup page, which is essential for user engagement and management. I would like to propose the creation of a streamlined signup page that allows users to register quickly and easily. Given that the site will primarily be used in urgent medical situations, the signup process should be simple and efficient.

## Database Options

To facilitate this, I suggest using either **MongoDB** or **Firebase**. Both options provide a user-friendly experience, ensuring that the signup process is quick and hassle-free.

## Key Considerations

- **Simplicity**: The signup form should only require essential information to minimize user friction. For example:
  ```javascript
  {
    "username": "string",
    "email": "string",
    "password": "string"
  }

Please @mansiruhil13 assign this task to me with a Level 3 tag; I would be happy to take it on. 

### Use Case

For smooth and fast onboarding

### Benefits

_No response_

### Priority

High

### Record

- [X] I agree to follow this project's Code of Conduct
- [X] I'm a GSSOC contributor
- [X] I want to work on this issue
- [X] I'm willing to provide further clarification or assistance if needed.

**Comment by mansiruhil13** [comment]
signup page already there??

**Comment by Saransh-Jainbu** [comment]
I don't think so as I couldn't find it, or the signup page button isn't redirecting ..
